build.rs

```diff
@@ -40,5 +48,60 @@ fn main() {
         println!("cargo:rustc-link-lib=static=chck-xdg");
         println!("cargo:rustc-link-lib=static=wlc-protos");
         println!("cargo:rustc-link-lib=static=wlc");
+    } else if let Err(s) = find() {
+        let _ = writeln!(io::stderr(), "{}", s);
+        process::exit(1);
+    }
+}
+
+fn find() -> Result<(), Error> {
+
+    let package_name = "wlc";
+    let shared_libs = ["wlc"];
+
+    // https://github.com/way-cooler/rust-wlc#rust-wlc currently
+    // (i.e. commit 97e0d5816fe7ec97dae20196d1e25c550f60f08f) suggests
+    // version 0.0.7.
+    let version = if cfg!(feature = "0.0.8") {
```

Comment:
`Cargo.toml` currently does not contain any reference to `wlc`.

I added the version checks because the `README.md` mentions the version interval, and
since the `pkg_config` crate does not seem to support the concept of "give me the latest
version -- whatever that currently may be", the checks were in order to implement the
interval.